### DISCUSSION TITLE
A question about generating mesh.

### DISCUSSION THREAD

**Discussion by BrownieLL** [idaholab/moose#25314] [Q&A Meshing]
Hello everyone,

I'm relatively new to Moose, and I'm really interested in developing a model that involves an inclined model domain (see below, noted that the vertical direction corresponds to the z-axis). I would like to inquire whether Moose has the capability to generate a grid for the inclined model domain. Your assistance would be greatly appreciated. Thank you in advance!
![Capture](https://github.com/idaholab/moose/assets/141759673/94212785-3cae-468e-85d5-97c51d3c8f40)

**Comment by GiudGiud** [comment]
Hello

We can do that.
Use a GeneratedMeshGenerator or CartesianMeshGenerator (second one will let you do this grading)
then a ParsedNodeTransformGenerator to slant it

https://mooseframework.inl.gov/docs/PRs/25087/site/source/meshgenerators/ParsedNodeTransformGenerator.html

Guillaume

**Reply by BrownieLL** [reply]
Thank you very much, Guillaume. That really helps!
